eventstore/mongodb/eventstore.go

```diff
@@ -94,6 +98,15 @@ func WithEventHandler(h eh.EventHandler) Option {
 	}
 }
 
+// WithCustomEventsCollectionName uses a different event collection than the default "events".
+func WithCustomEventsCollectionName(collName string) Option {
```

Comment:
This could have a shorter name. How about `WithCollectionName()`?

Reply:
Yes good point, kind of reminds me of some Java class names on second look 😄